### PR TITLE
Small modifications to Vote bot

### PR DESCRIPTION
**Pull request by pyko**
Had a look and I assume being able to have multi-word questions is actually not trivial? eg `/poll This is a question | Yes No Maybe`

### CODE REVIEW

**Comment by charleso** [comment]
> Had a look and I assume being able to have multi-word questions is actually not trivial? 

It's actually not that hard, just need to decide on the syntax to parse. It wouldn't affect the difficulty of the question. Did you want me to implement the above with the `|` separator?

File: src/Chat/Bot/Vote.hs [hunk 1]
```diff
@@ -52,16 +52,17 @@ createPoll question answers =
...
 --
 --   map :: (a -> b) -> [a] -> [b]
 --   unlines :: [String] -> String
+--   ++ :: [a] -> [a] -> [a]
 --
 pollRender :: Poll -> String
-pollRender poll =
-  notImplemented "Vote.pollRender" (pollRenderAnswer poll)
+pollRender (Poll question options) =
+  notImplemented "Vote.pollRender" (pollRenderAnswer (Poll question options))
```
Comment (charleso) [inline src/Chat/Bot/Vote.hs:68]: Just as aside you could do `p@(Poll question options)` and then refer to `p` as the `Poll`, but let's not go there. :)

Comment (pyko) [inline src/Chat/Bot/Vote.hs:68]: Ah, was wondering if there was a nice way to do that. Now I know :)

**Comment by pyko** [comment]
I think it'll be nice to be able to enter multi-word questions. Plus I'm now interested in how it is done (played around with the `Run/Vote.hs` but couldn't get it to work)

**Pull request merged**